infra/base-images/base-runner/Dockerfile

```diff
@@ -14,8 +14,15 @@
 #
 ################################################################################
 
+# Using multi-stage build to copy some LLVM binaries needed in the runner image.
+FROM gcr.io/oss-fuzz-base/base-clang AS base-clang
+
 FROM gcr.io/oss-fuzz-base/base-image
-MAINTAINER mike.aizatsky@gmail.com
```

Comment:
Thanks for cleaning this, can you remove the other maintainers from infra/ as well

```
base-images/base-builder/Dockerfile:MAINTAINER mike.aizatsky@gmail.com
base-images/base-clang/Dockerfile:MAINTAINER mike.aizatsky@gmail.com
base-images/base-image/Dockerfile:MAINTAINER mike.aizatsky@gmail.com
base-images/base-msan-builder/Dockerfile:MAINTAINER ochang@google.com
base-images/base-runner-debug/Dockerfile:MAINTAINER mike.aizatsky@gmail.com
base-images/base-runner/Dockerfile:MAINTAINER mike.aizatsky@gmail.com
base-images/msan-builder/Dockerfile:MAINTAINER ochang@google.com
```

Reply:
Done!